docs/tutorials/authentication.md

```diff
@@ -2,14 +2,78 @@
 
 There are multiple ways one can login to Synapse. We recommend users to choose the method that fits their workflow.
 
+## Prerequisites
+
+* Create a [personal access token](https://help.synapse.org/docs/Managing-Your-Account.2055405596.html#ManagingYourAccount-PersonalAccessTokens) token obtained from synapse.org under your Settings.
+    * Note that a token must minimally have the *view* scope to be used with the Synapse Python Client.
```

Comment:
Not sure if we wanted to outline that "view" scope won't be able to upload data into synapse (especially for the tutorials)

Reply:
Sure. Good idea. I'll mention this